specification/billing/resource-manager/Microsoft.Billing/stable/2020-11-01/invoiceSection.json

```diff
@@ -0,0 +1,469 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-11-01",
+    "title": "BillingManagementClient",
+    "description": "Billing client provides access to billing resources for Azure subscriptions."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/Microsoft.Billing/operations": {
```

Comment:
Let's not have this here as it may conflict with other version when we generate sdk using multiple versions

Reply:
Removed.